src/curated-corpus/components/ScheduleHistory/ScheduleHistory.styles.tsx

```diff
@@ -0,0 +1,21 @@
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+
+/**
+ * Styles for the ScheduleHistory component.
+ */
+export const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    collapse: {
+      marginTop: '0.5rem',
+      border: `solid 1px ${theme.palette.secondary.light}`,
+      borderRadius: '5px',
+      width: '100%',
+    },
+    scheduledHistoryWrapper: {
+      margin: '0.5em',
+    },
+    toggleHistoryButton: {
+      paddingLeft: '0px',
+    },
```

Comment:
@nina-py I've managed to have everything aligned to the left like we had it before I moved stuff from `<Box>` to `<Grid>` except the button itself.

I noticed that the actual button was aligned properly but it was the text inside which had some padding and gave that look of being a bit off from the left. I've removed the `padding-left` for now to fix this but the button does look a bit off when clicked. Open to suggestion 😄

Reply:
I think this is good enough! 👍  If it bugs either of us in the future, we can revisit.